demo/bnd-workspace/tycho.demo.markdown/api.bnd

```diff
@@ -0,0 +1 @@
+Export-Package: com.synesty.utils.markdown.api
```

Comment:
This looks like the wrong package name, I also have the problem that the resulting bundle is completely empty unless I put something similar to:

`-includeresource: ${target-dir}/classes`